src/components/subscribe/subscribe.scss

```diff
@@ -51,13 +51,16 @@
   }
 
   ///
-  /// 1. Allows for the visually hidden form to be keyboard accessible, will be
-  ///    visually hidden when elements within the form are not in focus.
+  /// 1. `:not(.activate-form)` allows for the visually hidden form to be
+  ///    keyboard accessible, will be visually hidden when elements within
+  ///    the form are not in focus.
   ///
 
+  // stylelint doesn't get that it can't collapse this nested `:not` selector.
+  // stylelint-disable selector-not-notation
   .c-subscribe__form-input-label,
-  .c-subscribe:not(.activate-form) // 1
-    &:not(:target):not(:focus-within) {
+  .c-subscribe:not(.activate-form) &:not(:target):not(:focus-within) {
```

Comment:
I'm not sure I follow. Can we not collapse `:not(:target):not(:focus-within)` to `:not(:target, :focus-within)`.

At a glance those seem the same to me?

Reply:
Oh yes, we can collapse that — The auto-fix was trying to collapse it to `.c-subscribe:not(.activate-form, :target, :focus-within)` which is not the same.

Reply:
That makes sense :+1: